src/System.Security.Cryptography.Csp/tests/RC2CryptoServiceProviderTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Security.Cryptography.Encryption.RC2.Tests
+{
+    using RC2 = System.Security.Cryptography.RC2;
+
+    public static partial class RC2CryptoServiceProviderTests
+    {
+        [Fact]
+        public static void RC2KeySize()
+        {
```

Comment:
This test doesn't seem specific to the CSP instance... why isn't it in the shared tests?